spec/features/user_can_pick_joke_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+feature "when I visit home page" do
+  fixtures :jokes
+  before(:each) do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.